### ISSUE TITLE
Textarea doesn't react to `cols` props

### ISSUE THREAD

**Issue by tombohub** [chakra-ui/chakra-ui#7165]
### Description

When I change `cols` props I expect textarea to rezise with new `cols`, but nothing happens.

### Link to Reproduction

https://codesandbox.io/s/happy-lumiere-df1905?file=/src/App.js

### Steps to reproduce

1. Go to code `Textarea` tag
2. Change `cols` props value
3. Nothing happens

### Chakra UI Version

2.4.2

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Hi @tombohub,

The `Textarea` component is consistent with how the native textarea works; check here:
https://codesandbox.io/s/bitter-snowflake-j27nvw?file=/src/App.js

If you're looking for an auto-resize functionality, that is not covered by the Textarea. You'll need to use `react-textarea-autoresize` or similar to achieve that.

**Comment by tombohub** [comment]
I don't understand your answer. You have put style width: 100% and claim it acts like native textarea.
Remove that style and it's obvious it's not consistent with native textarea: https://codesandbox.io/s/brave-tree-w05k06?file=/src/App.js

Thanks
